src/main/resources/db/baseline/B1_29__baseline.sql

```diff
@@ -0,0 +1,86 @@
+CREATE TABLE supplier
```

Comment:
Do the B prefixes work? I thought you needed to pay for flyway enterprise to get that